gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageClientImpl.java

```diff
@@ -105,6 +107,36 @@ public WritableByteChannel create(StorageResourceId resourceId, CreateObjectOpti
     return channel;
   }
 
+  @Override
+  public SeekableByteChannel open(
+      StorageResourceId resourceId, GoogleCloudStorageReadOptions readOptions) throws IOException {
+    logger.atFiner().log("open(%s, %s)", resourceId, readOptions);
+    return open(resourceId, null, readOptions);
```

Comment:
nit: add comment w/ parameter name for `null` parameter: `/* itemInfo= */ null,`